### PR TITLE
Limit BUS grant cap per annum

### PR DESCRIPTION
**Pull request by charlotte-avery**

### CODE REVIEW

File: simulation/costs.py [hunk 1]
```diff
@@ -345,20 +362,36 @@ def estimate_extended_boiler_upgrade_scheme_grant(
...
-        >= boiler_upgrade_funding_cap_gbp
+        >= BOILER_UPGRADE_SCHEME_GRANT_CAP[2024] / model_population_scale
     ):
         return 0
 
+    for year in BOILER_UPGRADE_SCHEME_GRANT_CAP.keys():
+        boiler_upgrade_funding_cap_gbp = (
+            BOILER_UPGRADE_SCHEME_GRANT_CAP[year] / model_population_scale
+        )
+        if (
```
[the remark this answers is not in the document]
Comment (shengy90) [reply simulation/costs.py:376]: IDK how govt works, but in a corporate world if budget is underspent, it usually doesn't get carried forward!

**Review by shengy90 — APPROVED** [review]
LGTM - approving with a clarifying comment

File: simulation/costs.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (charlotte-avery) [reply simulation/costs.py:376]: Yes, it get carried forward, I will add a clarifying comment :)

**Review by shengy90 — APPROVED** [review]
LGTM

**Pull request merged**